### PR TITLE
Roles of org [ User Role (Task #202199), System Role (Task #203308), Content (Task #203314)  and It's Count ]

### PR DESCRIPTION
**Pull request by poojakarma**
**Steps for Implementation of User Type and It's Count**
1. Create One column for show user Roles Icon
2. Get all user Type of Organisation using API
3. After getting all User type pass each user type in API and get count from another API
4. Integrated Count in userType list 
6. Show dialog box on click of button

**Steps for Implementation of System Role and It's Count**
1. create one more tab in popup window to show System Role and Count
2. Define constant system role and passing one by one system role in API to get Count
3. Show on UI

**Steps for Implementation of Content Type and It's Count**
1. Create new tab to see content and it's count
2. Define one function in ts to get Content Type and it's count of specific organisation
3. Show in UI

Link : https://drive.google.com/file/d/1PwTwoiZa0GkZNwcandRfj0Hr5Jqdj3kF/view?usp=drive_link

### CODE REVIEW

**Review by sanjeev-scm — APPROVED** [review]
The user type count API is called for each user type in the organisation as there is no support for the API

**Review by sanjeev-scm — APPROVED** [review]
Ready for merge

**Review by sanjeev-scm — APPROVED** [review]

**Review by sanjeev-scm — APPROVED** [review]
Can be merged

File: src/locales/en/en.json [hunk 1]
```diff
@@ -110,6 +111,14 @@
   "USER_SEARCH_EMAIL":"Search Email",
   "USER_SEARCH_PHONE":"Search Phone",
 
+  "PROFILE_ROLE":"User Role",
```
Comment (manojLondhe) [inline src/locales/en/en.json]: add screen/entity names prefix to all lang strings

File: src/locales/en/en.json [hunk 2]
```diff
@@ -10,11 +10,13 @@
...
   "NUMBER_OF_SUB_ORG":"Number of Sub-Organizations",
   "NUMBER_OF_USER_COUNT":"Number of Users",
-  "ORGANIZATION_SEARCH": "Search Organisation",
+  "ORGANIZATION_SEARCH": "Search",
   "ORGANIZATION_TOTAL":"Total Organizations",
   "SUB_ORGANIZATION_TOTAL":"Total Sub-organizations",
   "TOTAL_USER":"Total Users",
-  "ORGANIZATION_LIST_WARNING": "Organisation Data Not found",
+  "ORGANIZATION_LIST_WARNING":"Organisation Not Found",
+  "ROLE_INFO": "Roles Info",
```
Comment (manojLondhe) [inline src/locales/en/en.json]: add screen/entity names prefix to all lang strings

File: src/config/url.config.json [hunk 3]
```diff
@@ -8,6 +8,8 @@
         "USER_BLOCK_URL": "api/user/v1/block",
         "USER_UNBLOCK_URL": "api/user/v1/unblock",
         "USER_ROLES_GET_URL":"api/data/v1/form/read",
-        "USER_SEARCH_URL": "api/user/v1/search"
+        "USER_SEARCH_URL": "api/user/v1/search",
+        "USER_CONTENT_TYPE_COUNT" :"action/composite/v3/search"
```
Comment (manojLondhe) [inline src/config/url.config.json]: rename as `composite search url`

File: src/app/sb-admin/components/pages/sb-organization/sb-organization.component.ts [hunk 4]
```diff
@@ -134,7 +145,7 @@ export class SbOrganizationComponent implements OnDestroy {
       })
     },
       (error: any) => {
-        console.log(error);
+        console.log(error)
```
Comment (manojLondhe) [inline src/app/sb-admin/components/pages/sb-organization/sb-organization.component.ts]: remove all console logs

File: src/app/sb-admin/components/pages/sb-organization/sb-organization.component.ts [hunk 5]
```diff
@@ -64,7 +69,13 @@ export class SbOrganizationComponent implements OnDestroy {
       } else {
         this.loading = false;
       }
-    });
+    },
+      (error: any) => {
+        this.loading = false;
+        this.messageService.add({ severity: 'error', summary: "Oops! Something went wrong. Please try again later" })
+        console.log(error);
```
Comment (manojLondhe) [inline src/app/sb-admin/components/pages/sb-organization/sb-organization.component.ts]: remove all console logs

**Review by manojLondhe — CHANGES_REQUESTED** [review]

**Pull request merged**